### PR TITLE
[office-js] [office-js-preview] (Common API) Updating platform availability note for Office.CoercionType.SlideRange

### PR DESCRIPTION
**Pull request by alison-mk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: Reported in customer issue: https://github.com/OfficeDev/office-js-docs-reference/issues/1419
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/office-js/index.d.ts [hunk 1]
```diff
@@ -4440,7 +4440,7 @@ declare namespace Office {
          *   </tr>
          *   <tr>
          *     <td><code>Office.CoercionType.SlideRange</code></td>
-         *     <td>PowerPoint on the web and Windows</td>
+         *     <td>PowerPoint on the web and on Windows</td>
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:4443]: Consider formatting the applications in this table to look like the equivalent in office-js-preview
```suggestion
         *     <td>- PowerPoint on the web and on Windows</td>
```

Comment (alison-mk) [reply types/office-js/index.d.ts:4443]: Done!

File: types/office-js/index.d.ts [hunk 2]
```diff
@@ -4428,31 +4428,31 @@ declare namespace Office {
          *   </tr>
          *   <tr>
          *     <td><code>Office.CoercionType.Html</code></td>
-         *     <td>Word</td>
+         *     <td>- Word</td>
          *   </tr>
          *   <tr>
          *     <td><code>Office.CoercionType.Matrix</code> (array of arrays)</td>
-         *     <td>Excel and Word</td>
+         *     <td>- Excel and Word</td>
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:4435]:
```suggestion
         *     <td>- Excel<br>- Word</td>
```

File: types/office-js/index.d.ts [hunk 3]
```diff
@@ -4428,31 +4428,31 @@ declare namespace Office {
...
          *   </tr>
          *   <tr>
          *     <td><code>Office.CoercionType.SlideRange</code></td>
-         *     <td>PowerPoint on the web and Windows</td>
+         *     <td>- PowerPoint on the web and on Windows</td>
          *   </tr>
          *   <tr>
          *     <td><code>Office.CoercionType.Table</code> (TableData object)</td>
-         *     <td>Excel and Word</td>
+         *     <td>- Excel and Word</td>
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:4447]:
```suggestion
         *     <td>- Excel<br>- Word</td>
```

File: types/office-js/index.d.ts [hunk 4]
```diff
@@ -4428,31 +4428,31 @@ declare namespace Office {
...
          *   </tr>
          *   <tr>
          *     <td><code>Office.CoercionType.Text</code> (string)</td>
-         *     <td>Excel, PowerPoint, Project, and Word</td>
+         *     <td>- Excel, PowerPoint, Project, and Word</td>
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:4451]:
```suggestion
         *     <td>- Excel<br>- PowerPoint<br>- Project<br>- Word</td>
```

File: types/office-js/index.d.ts [hunk 5]
```diff
@@ -4428,31 +4428,31 @@ declare namespace Office {
...
          *   </tr>
          *   <tr>
          *     <td><code>Office.CoercionType.XmlSvg</code></td>
-         *     <td>Excel on Windows and Mac, PowerPoint on Windows, Mac, and the web, and Word on Windows and Mac</td>
+         *     <td>- Excel on Windows and Mac, PowerPoint on Windows, Mac, and the web, and Word on Windows and Mac</td>
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:4455]:
```suggestion
         *     <td>- Excel on Windows and on Mac<br>- PowerPoint on Windows, on Mac, and on the web<br>- Word on Windows and on Mac</td>
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**